apps/condo/domains/onboarding/hooks/TourContext/useCompletedTourModals.tsx

```diff
@@ -358,47 +372,51 @@ export const useCompletedTourModals = ({ activeStep, setActiveTourStep, refetchS
                     </Button>,
                 ]}
             >
-                <Space size={40} direction='vertical'>
-                    <NewAbilitiesWrapper>
-                        <Row>
-                            {
-                                !isEmpty(get(computedCompletedStepModalData, 'newEmployeeFeatures')) && (
-                                    <Col span={12}>
-                                        <Space size={20} direction='vertical'>
-                                            <Typography.Title level={4}>{NowYouCanMessage}</Typography.Title>
-                                            <Space size={16} direction='vertical'>
-                                                {
-                                                    computedCompletedStepModalData.newEmployeeFeatures.map(label => (
-                                                        <UnlockedFeature key={label} label={label} />
-                                                    ))
-                                                }
+                <Row gutter={[0, 40]}>
```

Comment:
why change it to row/col?
other complete modal steps still works normally?

Reply:
select didn't scale to full width, Col does